aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -223,6 +223,8 @@ See Visual Studio Code.
 
 ---
 
+Enabling XML comments provides debug information for undocumented public methods: __Missing XML comment for publicly visible type or member__
```

Comment:
I don't understand 
`__Missing XML comment for publicly visible type or member__`

Reply:
That is the error that shows up when  you start the debugger by pressing f5 when you don't have all of your methods with XML comments (or at least that is the error I was getting in VS code)